Kajabi/sage-lib
# [Sage-377] Modal/Drawer - Fix Scrolling with Nested Sage Row

## Description

Fixes an issue where a Sage Row causes horizontal scrolling on desktop.


## Testing in `sage-lib`

- View [Contact Profile Mock](http://localhost:4100/?path=/story/mocks-contact-profile-full-view--default)
- Check that the component cannot be scrolled horizontally.

## Testing in `kajabi-products`

(**LOW**) Legacy/Next - Fixes an issue where a Sage Row causes horizontal scrolling on desktop.


## Related

[Sage-377](https://kajabi.atlassian.net/browse/SAGE-377)
